protos/arflow/service.proto

```diff
@@ -55,7 +59,7 @@ message RegisterRequest {
 }
 
 message RegisterResponse {
-    string message = 1;
+    string uid = 1;
```

Comment:
Why do we want to change the `message` field to `uid`? To align with other messages?

Reply:
Yes, that's what I'm thinking

Reply:
I see. In that sense, we probably want to change the `message` field in `DataFrameResponse` to `uid` as well.

Reply:
Looking into the code, it seems like the only use of `DataFrameResponse` is for the server to ACK the client that a data frame has been received. I think we should keep it as is

```python
class ARFlowService(...):
    
    def data_frame(...):
        ...
        return service_pb2.DataFrameResponse(message="OK")
```

Reply:
Oh, I realized the `uid` actually means session generated by the registration request. While this message field can mean anything. Sure, we can keep the message field in the `DataFrameResponse` untouched.